CodeForAfrica-SCRAPERS/opengazettes_tz_scrapy
# Scraper for Tanzania Government Gazettes

**What does this PR do?**

- Adds two spiders for two _different_ domains
- Adds an Item Pipeline for download and storage of file
- Stores the gazettes in a specified directory using the specified structure eg. Year > Month
- Directs the feed exports to an online s3 bucket


**How can this be manually tested**

There is a job on scrapinghub to scrape the TZ gazettes

![imageedit_5_7953122664](https://user-images.githubusercontent.com/25247565/29110116-3dfb2c3c-7cee-11e7-91f2-2080fbe19efa.jpg)

- step 1: Choose the correct spider to run (`tz_gazettes_v2` for now)
- step 2: Add the year of gazettes wanted to be scraped as an `Argument`
- step 3: Run the job

![imageedit_16_3019657827](https://user-images.githubusercontent.com/25247565/29110137-4b9932d0-7cee-11e7-8d80-75107e103126.jpg)

Check its s3 bucket to find the scrapped gazettes
@DavidLemayian
